srv/www/regionService/regionInfo.py

```diff
@@ -156,8 +156,12 @@ def create_smt_region_map(conf):
             try:
                 ipaddress.IPv6Network(ip_range)
             except ValueError:
-                msg = 'Could not proces range, improper format: %s'
-                logging.error(msg % ip_range)
+                if ip_range:
```

Comment:
My question is why we would tolerate a broken configuration file. This if condition allows, for example configuration file that has

`public-ipsv6=,`
or 
`public-ipsv6=11:22:33:,44:55:66,`

In the first case why would there be a configuration entry when there are no values to configure and in the second case, the trailing comma is a syntax error.

Last but not least if we tolerate such broken config files for IPv6 why would we not tolerate it for IPV4.

I am not in favor of tolerating broken configuration files. Broken configuration files should trigger an error, which the current implementation does.

Reply:
The config files were fixed as well to not to have empty values (original error)

Reply:
I kept the same behaviour (right behaviour) but I think it is still good/valuable to have different error messages